src/main/kotlin/no/nav/familie/ba/sak/kjerne/steg/StegService.kt

```diff
@@ -60,39 +60,29 @@ class StegService(
     @Transactional
     fun håndterNyBehandling(nyBehandling: NyBehandling): Behandling {
         val behandling = behandlingService.opprettBehandling(nyBehandling)
-
-        return if (nyBehandling.behandlingType == BehandlingType.MIGRERING_FRA_INFOTRYGD || nyBehandling.behandlingÅrsak == BehandlingÅrsak.FØDSELSHENDELSE) {
-            håndterPersongrunnlag(
-                behandling,
-                RegistrerPersongrunnlagDTO(
-                    ident = nyBehandling.søkersIdent,
-                    barnasIdenter = nyBehandling.barnasIdenter
-                )
-            )
+        val barnasIdenter: List<String>
+        if (nyBehandling.behandlingÅrsak in listOf(BehandlingÅrsak.MIGRERING, BehandlingÅrsak.FØDSELSHENDELSE)) {
```

Comment:
```suggestion
        val barnasIdenter: List<String> = if (nyBehandling.behandlingÅrsak in listOf(BehandlingÅrsak.MIGRERING, BehandlingÅrsak.FØDSELSHENDELSE)) {
```

Har dere endret på BehandlingÅrsak? Ser at vi håndterer migrering og fødselshendelse likt, men hadde migrerning_fra_infotrygd og fødselshendelser.